common/djangoapps/enrollment/api.py

```diff
@@ -177,11 +188,18 @@ def add_enrollment(user_id, course_id, mode='honor', is_active=True):
             }
         }
     """
+
     _validate_course_mode(course_id, mode)
-    return _data_api().create_course_enrollment(user_id, course_id, mode, is_active)
+    return _data_api().create_course_enrollment(user_id, course_id,
+            mode, is_active)
 
 
-def update_enrollment(user_id, course_id, mode=None, is_active=None):
+def update_enrollment(
+    user_id,
+    course_id,
+    mode=None,
+    is_active=None,
+    ):
```

Comment:
In other words, please update your Sublime configuration and revert the line length changes in the above lines :)